runtime/doc/filetype.txt

```diff
@@ -510,63 +510,83 @@ Local mappings:
 	to the end of the file in Normal mode.  This means "> " is inserted in
 	each line.
 
+
 MAN					*ft-man-plugin* *:Man* *man.vim*
 
-View manpages in Nvim. Supports highlighting, completion, locales, and
-navigation. Also see |find-manpage|.
+View man pages in Nvim.  Supports highlighting, completion, locales, and
+navigation.  Also see |find-manpage|.
```

Comment:
We don't do double-spacing in our docs (at least not consistently, and certainly not for new docs!)